### ISSUE TITLE
Show build progress when buildOnSave

### ISSUE THREAD

**Issue by EmpireJones** [Microsoft/vscode-go#1869]
It looks like https://github.com/Microsoft/vscode-go/issues/1456 implemented showing "building..." in the status bar when building manually. This is a feature request to show this indication when a build is triggered via buildOnSave too.

**Comment by ramya-rao-a** [comment]
Hey @EmpireJones,

Apologies for taking so long to get to this issue.

Below are the reasons we chose not to use the status bar for the on save operation:
- Build is not the only thing that runs on save. We have lint and vet as well. Showing 3 status bar items is not a good idea
- Showing a single one that says something like "On Save operations running..." might work, but it would keep appearing and disappearing especially for users who have enabled the auto save feature, which can be distracting.

The one case where this can be useful is for long running build/lint/vet operations on save.

Currently, we use the Output channel to show the results of these processes, but it only shows up after the process completes.

We can update this to print a line that indicates the start of the process as well, so that users would know that the build process has started and continue to monitor the channel for completion.

I have pushed a commit to do exactly that. This will ship in our next update.
